lesson_04/homework_04.py

```diff
@@ -0,0 +1,104 @@
+adwentures_of_tom_sawer = """\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+"Big Missouri" worked ....
+and sweated
+in the sun,
+the retired artist sat on a barrel in the .... shade close by, dangled his legs,
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+By the time Ben was fagged out, Tom had traded the next chance to Billy Fisher for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+
+##  ПЕРЕЗАПИСУЙТЕ зміст змінної adwentures_of_tom_sawer у завданнях 1-3
+# task 01 ==
+""" Дані у строці adwentures_of_tom_sawer розбиті випадковим чином, через помилку.
+треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
+print("Task 01")
+adwentures_of_tom_sawer = adwentures_of_tom_sawer.replace("\n", " ")
+print(adwentures_of_tom_sawer)
+
+# task 02 ==
+""" Замініть .... на пробіл
+"""
+print("\nTask 02")
+adwentures_of_tom_sawer = adwentures_of_tom_sawer.replace("....", " ")
+print(adwentures_of_tom_sawer)
+
+# task 03 ==
+""" Зробіть так, щоб у тексті було не більше одного пробілу між словами.
+"""
+print("\nTask 03")
+adwentures_of_tom_sawer = " ".join(adwentures_of_tom_sawer.split())
+print(adwentures_of_tom_sawer)
+
+# task 04
+""" Виведіть, скількі разів у тексті зустрічається літера "h"
+"""
+print("\nTask 04")
+count = adwentures_of_tom_sawer.find("h")
+print(f"There are {count} times the letter \"h\" in the sentence \"adwentures_of_tom_sawer\".")
```

Comment:
count variable value will be the index of it`s existence in the string. If you want to count how much time "h" symbol is present in the string - you need to iterate for whole string and each time if symbol equal to "h" - increase counter by 1.

Reply:
Fixed ib a562dda4dc7cfd297643dcf1f5dff66a9ac76b0f